src/OperationResponse.php

```diff
@@ -340,4 +377,13 @@ public function getMetadata()
         $metadata->mergeFromString($any->getValue());
         return $metadata;
     }
+
+    private function operationsCall($method, $name, array $additionalArgs)
+    {
+        if (is_null($method)) {
+            throw new LogicException('cancelling operations is not supported by this API');
```

Comment:
This needs to throw an RPC error code for not implemented, to be consistent with OnePlatform

Reply:
On second thought, do we really want to throw an `ApiException` when no API call was made? This is misleading. Rather, what we have here seems appropriate.